docs/pages/faq.mdx

```diff
@@ -125,6 +125,10 @@ users to upgrade.
 This check is performed against all cluster components, including the Proxy
 Service and Auth Service, as well as agents running other Teleport Services.
 
+## What is the minimum TLS version that Teleport requires?
+
+TLS 1.2 is the minimum version that Teleport allows.
```

Comment:
I think another sentence or two of context would be good here. Does this mean that if you want an application to speak TLS with a Teleport process, it needs to use TLS 1.2? Are there any more specific problems that users run into regarding TLS versioning that we want to call out explicitly here?

Reply:
I don't think so. Usually folks asking this question are following one of those security checklist questionnaires and they can't find the answer in the docs.

Reply:
I still think this could be a bit more precise—does this mean that if you want an application to speak TLS with a Teleport process, it needs to use TLS 1.2 or higher?

Reply:
What about the following?

> ## What is the minimum TLS version that Teleport requires?
> 
> Teleport requires a minimum of TLS version 1.2.
> 
> This means that when applications and clients establish or accept TLS connections with Teleport processes, they must use TLS 1.2 or a higher protocol version. Teleport enforces this requirement in all operations that involve TLS connections.

Reply:
Looks great!